crates/core/src/kvs/node.rs

```diff
@@ -249,9 +250,13 @@ impl Datastore {
 					let end = crate::key::table::lq::suffix(&ns.name, &db.name, &tb.name);
 					let mut next = Some(beg..end);
 					while let Some(rng) = next {
-						let res = catch!(txn, txn.batch(rng, *NORMAL_FETCH_SIZE, true, None).await);
+						// Pause and yield execution
+						yield_now!();
```

Comment:
not sure if it's helpful to explicitly call `yield_now!()` right next to `.await`

Reply:
Apparently `await` (if it doesn't call any tokio specific function internally) doesn't yield back to the Tokio executor. Need to do more investigation here, but given that this is a background job, it's speed isn't a priority.